README.md

```diff
@@ -8,15 +8,68 @@ fresh and interesting. The main idea is that the script would download
 any JPEG or PNG formatted image that it found listed in the specified
 subreddit and download them to a folder.
 
+## jtara1 Fork
 
-# Requirements:
+### Features and Changes:
 
- * Python 2 (Python3 might be supported over 2to3, but see for
-   yourself and report back).
+
+
+* Adapted to Python 3 mostly by merge from [ohyou/RedditImageGrab](https://github.com/ohyou/RedditImageGrab) along with some additional fixes
+
+* \-\-num cli argument now counts by reddit submission rather than individual image
+
+    * added my fork (jtara1/imgur-downloader) to handle all imgur downloads (making the above feature possible)
+
+
+* file .\_history.txt contains reddit id of last downloaded and is identified by subreddit & ARGS.sort\_type, e.g.:
+
+    > {'wallpapers': {'topmonth': {'last\-id': '4x4so2'}}}
+
+* positional arguments, \<subreddit\> and \<dest\_file\>, changed to optional cli arguments
+
+    *    positional arguments changed
+
+    *    --dir is required
```

Comment:
it is possible. use ```os.path.isfile``` to check if input is filename and check if it have ```.txt``` extension. something similar like this

```python
if os.path.isfile(filename) and os.path.splitext(filename)[1] == '.txt':
```